inactiveLink.html

```diff
@@ -0,0 +1,21 @@
+<head>
```

Comment:
The concept behind the inactiveLink.html page will be changed per Dave's comment.  At the bottom of each chapter, we can incorporate temporary targets for to-be-created links, whether those links ultimately will target the same chapter or another chapter.